docs/build/quick-start.md

```diff
@@ -1,303 +1,138 @@
 
 !!! warning ""
+    **Want an even faster way to try Ceramic? Visit the [Playground Demo App](https://playground.ceramic.dev){:target="_blank"} to test the full stack of Ceramic Components in the browser.
 
-    **Want an even faster way to try Ceramic?** Visit the [Playground demo app](https://playground.ceramic.dev){:target="_blank"} to test the full stack of Ceramic components in the browser.
+## **1. Install the Client**
 
-## **1. Install the CLI**
-
-Visit the [Ceramic CLI](./cli/installation.md) page for instructions on how to quickly install the CLI.
+Visit the [Ceramic Client](./javascript/installation.md) page for instructions on how to quickly install the Client.
 
 ## **2. Create a stream**
-Use the `create` command to create a new [stream](../learn/glossary.md#streams). In the example below we create a stream that uses the [TileDocument StreamType](../streamtypes/tile-document/overview.md). Note that *TileDocument* is the only [StreamType](../learn/glossary.md#streamtypes) that can currently be created by the Ceramic CLI.
+Creating streams is dependent on the StreamType of the record. More details can be found in our [StreamTypes Overview](../streamtypes/overview.md). A basic sample can be found below.
 
 === "Command"
 
-    ```bash
-    $ ceramic create tile --content '{ "Foo": "Bar" }'
-    ```
+    ```JavaScript
+    import { TileDocument } from '@ceramicnetwork/stream-tile'
 
-=== "Output"
+    const doc = await TileDocument.create(ceramic, content, metadata, opts)
 
-    ```bash
-    StreamID(kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa)
-    {
-        "Foo": "Bar"
-    }
+    const streamId = doc.id.toString()
     ```
 
-    !!! quote ""
-        The first line of the output is the [StreamID](../learn/glossary.md#streamid), which is the persistent identifier of our newly created stream. This StreamID will be different for you, since you created it with your DID. Below the StreamID is the current content of the stream.
-
-??? info "More options"
-    
-    - `--controllers`: set the *controller* of the stream
-    - `--schema`: set the *schema* of the TileDocument
-    - Run `ceramic create -h` to see all available options
-
 ## **3. Query a stream**
-Use the `show` command to query the current [state](../learn/glossary.md#state) of a stream. You will need to provide its *StreamID*.
+Use the `loadStream()` function to query the current [state](../learn/glossary.md#state) of a stream. You will need to provide it's *StreamID*
 
 === "Command"
 
-    ```bash
-    $ ceramic show kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa
-    ```
-    
-    !!! quote ""
-        You should use your StreamID instead of the StreamID included here.
+    ```JavaScript
+    import CeramicClient from '@ceramicnetwork/http-client'
 
-=== "Output"
+    const ceramic = new CeramicClient("https://gateway-clay.ceramic.network")
```

Comment:
Here we show how to set up the client but above for the "create a stream" example we do not.  Whichever we do we should be consistent.